internal/run/run.go

```diff
@@ -141,7 +141,12 @@ func (w *Workflow) RunWithVisualization(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		tOut := "the current directory"
+		workingDirectory, err := os.Getwd()
```

Comment:
Note that `the current directory` is not helpful as we set the working directory to be the `outDir` inside quickstart.go, rather be explicit especially as all the history of responses of the forms are cleared from the buffer. Is it easy to make all the forms responses persist?

Reply:
Not that I know of

Reply:
I would just not change this in this PR

Reply:
That's fair enough. I can move it out into another PR

Reply:
Moved to https://github.com/speakeasy-api/speakeasy/pull/586